projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,88 @@
+# Using Store with Angular Elements
+## Creating Angular Elements
+The following recipe shows you how to manage the state of a counter, and how to select and display it within an Angular Elements and use the same within an Angular Project and a static page.
```

Comment:
```suggestion
The following recipe illustrates utilizing Angular Elements with a NgRx Store to manage the state of a counter, and select and display that counter state from within separate Angular Elements. All from within a single Angular Project and a static HTML page.
```

Reply:
@santoshyadav198613 one more change here...

`All from within a single Angular Project and a static HTML page.`
should be
`This is all accomplished from within a single Angular Project and static HTML page.`

Reply:
Done.